### DISCUSSION TITLE
Cancelled: Next GE meeting: Monday Feb 20?.. 🎭

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#101] [Team Posts]
It's the 🎭Carnival🎭 in some countries :). Including Spain but I'd still be willing to have a meeting, assuming the Brazil crown attends :) @arademaker @fcbond @danflick ? If this seems like not a good idea, we can skip this one. Let me know :).

**Comment by emilymbender** [comment]
It's President's Day here in the US, so I wouldn't be able to join.

**Reply by olzama** [reply]
@arademaker , @danflick , @fcbond ? Meeting or no meeting tomorrow, could you please confirm?

**Reply by arademaker** [reply]
I will not be able to make. Better skip this?

**Reply by olzama** [reply]
Yeah, I think so. I myself want to go do carnival things here, to be honest! Let's meet on March 6 then. Meanwhile though, I'd really appreciate if @danflick could maybe take a look at this: https://delphinqa.ling.washington.edu/t/treebanking-checking-across-decisions-preferences-results-and-relations/896 -- then I might be more sure that I am doing the right stuff in the next two weeks!
